### PR TITLE
Allow adding description to section of any type

### LINKED ISSUE #360: Allow adding description to section of any type
Right now `section.description` field can only be set for product finals. Use case - adding links to conference calls with the candidate

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Feature

## Related issues

Resolve #360

### CODE REVIEW

**Comment by Katrin-kudryash** [comment]
https://github.com/taskany-inc/hire/assets/81121562/74e4a7e4-249e-4b32-a402-b20863ef1961

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx [hunk 1]
```diff
@@ -189,28 +194,32 @@ export const CreateOrUpdateSectionForm = ({
...
-                                <FormInput
-                                    {...register('description')}
-                                    label={tr('Description')}
-                                    error={errors.description}
-                                    placeholder={tr('In which team is the product final held?')}
-                                    autoComplete="off"
-                                    flat="bottom"
-                                />
-                            )}
+                            <FormInput
```
Comment (tsumo) [inline src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx:211]: Can we change it to text area?

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx [hunk 2]
```diff
@@ -189,28 +194,32 @@ export const CreateOrUpdateSectionForm = ({
...
+                                {...register('description')}
+                                label={tr('Description')}
+                                error={errors.description}
+                                placeholder={
+                                    sectionType.userSelect
+                                        ? tr('Which team is held the interview?')
+                                        : tr('Add a link to jazz or a couple of notes')
```
Comment (tsumo) [inline src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx:218]: ссылку на встречу / link to a meeting

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx (hunk 1 above)
Comment (nyet-ty) [reply src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx:211]: +

**Comment by Katrin-kudryash** [comment]
Untitled 7
Untitled 8

File: src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.i18n/ru.json [hunk 3]
```diff
@@ -4,5 +4,6 @@
     "Calendar": "Календарь",
     "Save the section": "Сохранить секцию",
     "Description": "Описание",
-    "In which team is the product final held?": "В какую команду проводится продуктовый финал?"
+    "In which team is the product final held?": "",
```
Comment (nyet-ty) [inline src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.i18n/ru.json:7]: I think you missed translation here

**Review by nyet-ty — APPROVED** [review]

**Review by tsumo — CHANGES_REQUESTED** [review]
```
Check warning on line 113 in src/components/CreateOrUpdateSectionForm/CreateOrUpdateSectionForm.tsx

React Hook useCallback has a missing dependency: 'schedulable'. Either include it or remove the dependency array
```

**Review by tsumo — APPROVED** [review]

**Pull request merged**